.circleci/config.yml

```diff
@@ -0,0 +1,73 @@
+---
+x-defs:
+  default_docker: &rust_docker [{image: "rustlang/rust:nightly"}]
+
+version: 2
+jobs:
+  test:
+    docker: *rust_docker
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - ds-v1-docker--{{ arch }}-{{ checksum "Cargo.lock" }}-{{ epoch }}
+            - ds-v1-docker--{{ arch }}-{{ checksum "Cargo.lock" }}-
+            - ds-v1-docker--{{ arch }}-
+      - run: cargo fetch
+      - run: cargo build
+      - save_cache:
+          key: ds-v1-docker--{{ arch }}-{{ checksum "Cargo.lock" }}-{{ epoch }}
```

Comment:
Isn't this always going to save a new cache, even if the contents haven't changed, because of {{epoch}}? Can we just drop {{epoch}} from all of these keys?

Reply:
I'd still like to remove `{{epoch}}` from all of these cache keys.

Reply:
My reasoning for keeping this this way was that I don't want to assume that `{{ checksum "Cargo.lock" }}` is the only thing that will change, because it's not, right?

Right?

- - -

Well, actually, it sure looks like it is definitive.

Okay, fair. I'll nix it.